src/PlanningBiblio/Helper/HolidayHelper.php

```diff
@@ -121,10 +122,41 @@ public function getCountedHours()
 
             $debutConges = $current == $debut ? $hre_debut : "00:00:00";
             $finConges = $current == $fin ? $hre_fin : "23:59:59";
+
+            // Convert free break as fixed break if needed
+            $free_break_already_removed = false;
+            $has_fixed_break = isset($planning['times'][$day_id][1]) && $planning['times'][$day_id][1] != "";
```

Comment:
You should also check $planning['times'][$day_id][5] : second pause when the setting pause2 is enabled.

Can this test be replaced by $has_fixed_break = !empty($planning['times'][$day_id][1]); ?